layers/+spacemacs/spacemacs-defaults/keybindings.el

```diff
@@ -1053,3 +1053,70 @@ If FRAME is nil, it defaults to the selected frame."
   'spacemacs/scale-transparency-transient-state/spacemacs/toggle-transparency)
 
 ;; end of Transparency Transient State
+
+;; Background Transparency transient-state
+
+(defun spacemacs/enable-background-transparency (&optional frame alpha-background)
+  "Enable background transparency for FRAME.
+If FRAME is nil, it defaults to the selected frame.
+ALPHA is a pair of active and inactive transparency values. The
+default value for ALPHA is based on "
```

Comment:
I think something has been cut of in your comment.

Reply:
Fixed